### PR TITLE
recognize macOS 11.1 as big sur

### PR DESCRIPTION
**Pull request by aumuell**
beta 11.1 is still big sur

### CODE REVIEW

**Review by adamjstewart — APPROVED** [review]
I just noticed this announcement as well.

**Comment by adamjstewart** [comment]
@alalazo can we monkeypatch `platform.mac_ver` just to get full coverage of this file? It would be good to add tests to ensure that `10.15.X` is Catalina and `11.X.Y` is Big Sur.

**Comment by tgamblin** [comment]
There's a simpler way to get the coverage (though it's kind of cheating).  See my latest commit.

File: lib/spack/spack/operating_systems/mac_os.py [hunk 1]
```diff
@@ -55,10 +55,14 @@ def __init__(self):
             '10.14': 'mojave',
             '10.15': 'catalina',
             '10.16': 'bigsur',
-            '11.0':  'bigsur',
+            '11':  'bigsur',
         }
 
-        mac_ver = str(macos_version().up_to(2))
+        # Big Sur versions go 11.0, 11.0.1, 11.1 (vs. prior versiosn that
```
Comment (adamjstewart) [inline lib/spack/spack/operating_systems/mac_os.py]: Typo

**Review by adamjstewart — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Pull request merged**